scipy/stats/_continuous_distns.py

```diff
@@ -7964,6 +7965,136 @@ def _sf(self, x, chi):
         """
         return _argus_phi(chi * np.sqrt(1 - x**2)) / _argus_phi(chi)
 
+    def _rvs(self, chi):
+        chi = np.asarray(chi)
+        if chi.size == 1:
+            if chi <= 26:
```

Comment:
I'd like to see some reasoning in the comments for choosing `26` as the changeover value.  `np.exp(chi**2 / 2)` doesn't overflow until `chi>=37.67`.